### ISSUE TITLE
Is this package maintained anymore?  

### ISSUE THREAD

**Issue by bryanbassett** [GetStream/stream-laravel#92]
I am running the latest laravel and I even followed the demo projects examples for some tips. But when I now try to create a new item that belongsTo the user which hasMany of the item(s), I get this error:

BadMethodCallException with message 'Call to undefined method Illuminate\Database\Eloquent\Relations\BelongsTo::getKey()

If this package is no longer maintained I might just have to write the implementation myself instead of using the nice eloquent functionality it seems you all had worked hard to make. 

On the other hand it could just be me, and that is also very likely. I have no idea why it won't work. I can create a new "item" if I dont use the ActivityTrait in the model.

**Comment by ferhatelmas** [comment]
Yes, it's but this is possible since quite old at the moment. Luckily, revamp is coming soon. I will drop an update here by Monday.

**Comment by bryanbassett** [comment]
![image](https://user-images.githubusercontent.com/3886449/119170065-be5e9c00-ba30-11eb-8ef2-cc647e05e623.png)

I was able to solve my issue by using these overrides. Overflows? Idk what they are technically called. But yeah. Solved my issue at least. I still think this should not be necessary however. I think the problem was that get_class($instance) was returning BelongsTo:: instead of the actual class name of the model.

**Comment by dgillier** [comment]
@ferhatelmas Hello any news about a package update (was supposed to be posted in May) ?
Thanks, Denis

**Comment by dgillier** [comment]
Any update ?

**Comment by ferhatelmas** [comment]
FYI, there is a new release https://github.com/GetStream/stream-laravel/releases/tag/2.4.0
